docs/aws.rst

```diff
@@ -1,10 +1,78 @@
 Amazon Web Services
 ===================
 
+Parsons provides utility functions and/or connectors for three different `AWS services<https://aws.amazon.com/>`_:
+
+* :ref:`_aws_lambda`: AWS's `serverless computing platform <https://aws.amazon.com/lambda/>`_
+* :ref:`_aws_s3`: AWS's `object storage service <https://aws.amazon.com/s3/>`_
+* :ref:`_redshift`: AWS's `data warehousing service <https://aws.amazon.com/redshift/>`_, with two additional classes providing utility functions:
+  * :ref:`_redshift_table_and_view_api`: Methods for managing tables and views
+  * :ref:`_redshift_schema_api`: Methods for managing schema
```

Comment:
Not sure that it matters, but there is a little bit of whitespace here.

Reply:
My goal was for this to be sub bullet points. I Googled how to format lists and I think this is right, but I'm not set up to test the docs locally.